wremnants/theory_tools.py

```diff
@@ -155,15 +155,14 @@ def define_and_make_pdf_hists(df, axes, cols, dataset, pdfset="nnpdf31", storeUn
     tensorASName = f"{pdfName}ASWeights_tensor"
     entries = pdfInfo["entries"] if storeUnc else 1
 
+
     df = df.Define(tensorName, f"auto res = wrem::clip_tensor(wrem::vec_to_tensor_t<double, {entries}>({pdfBranch}), 10.); res = nominal_weight/nominal_pdf_cen*res; return res;")
 
     pdfHist= df.HistoBoost(pdfName, axes, [*cols, tensorName])
 
-    df = df.Define(tensorASName, "Eigen::TensorFixedSize<double, Eigen::Sizes<2>> res; "
-            f"res(0) = {pdfInfo['alphas'][0]}; "
-            f"res(1) = {pdfInfo['alphas'][1]}; "
-            "return wrem::clip_tensor(res, 10.)")
-    alphaSHist = df.HistoBoost(f"alphaS002{pdfName}", axes, [*cols, tensorASName])
+    # slice 2 elements starting from 101
```

Comment:
Don't think this is an intended change either. Probably best to make the change to include a name for the pdf hist on top of the current main (could be a separate PR)